### ISSUE TITLE
Duplicate variable error

### ISSUE THREAD

**Issue by edba** [systemjs/builder#512]
Using systemjs-plugin-babel will throw an error when a function contains a variable with the same name as the function.

Error: **Unhandled rejection TypeError: Cannot read property 'reportError' of null**

Example:

test.js
```javascript
'use strict';

const test = {
  echo: function () {
    const echo = '';
  }
};

export default test;
```

package.json
```json
{
  "devDependencies": {
    "systemjs-builder": "0.15.10",
    "systemjs-plugin-babel": "0.0.7"
  }
}
```
system.config.js
```javascript
"use strict";
System.config({
  "baseURL":  "/",
  "transpiler": "plugin-babel",
  "map": {
    "plugin-babel": "node_modules/systemjs-plugin-babel/plugin-babel.js",
    "systemjs-babel-build": "node_modules/systemjs-plugin-babel/systemjs-babel-browser.js"
  }
});
```

build.js
```javascript
var Builder = require('systemjs-builder');
var builder = new Builder('./','system.config.js');

builder.config({
  'map': {
    'systemjs-babel-build': 'node_modules/systemjs-plugin-babel/systemjs-babel-node.js'
  }
});

builder.bundle('test.js', 'test-bundle.js');
```

run: **npm install; node build.js**

**Comment by guybedford** [comment]
This looks like a Traceur bug - I've posted https://github.com/google/traceur-compiler/issues/2091.

**Comment by guybedford** [comment]
This is resolved in the latest version of Traceur, which will come with the next release of SystemJS Builder.

**Comment by guybedford** [comment]
Released in 0.15.14.
